std/StdTypes.hx

```diff
@@ -26,8 +26,10 @@
 
 	@see https://haxe.org/manual/types-void.html
 **/
-#if jvm @:runtimeValue #end
-@:coreType abstract Void { }
+#if jvm
+@:runtimeValue
+#end
```

Comment:
I think these should also be kept as single-line

Reply:
Probably the same as https://github.com/HaxeCheckstyle/haxe-formatter/issues/332.